src/rpc/core_rpc_server.cpp

```diff
@@ -919,6 +1008,10 @@ namespace cryptonote
   bool core_rpc_server::on_getblockcount(const COMMAND_RPC_GETBLOCKCOUNT::request& req, COMMAND_RPC_GETBLOCKCOUNT::response& res)
   {
     PERF_TIMER(on_getblockcount);
+    // if (should_use_bootstrap_daemon())
+    // {
+    //   return use_bootstrap_daemon_json<COMMAND_RPC_GETBLOCKCOUNT>("getblockcount", req, res)) return true;   // TODO: doesn't work because COMMAND_RPC_GETBLOCKCOUNT::request is typedef'd as std::list<s>
+    // }
```

Comment:
The above code doesn't compile since `COMMAND_RPC_GETBLOCKCOUNT` probably because `COMMAND_RPC_GETBLOCKCOUNT::request` is just a `typedef` of `std::list<std::string>` and doesn't have the `BEGIN_KV_SERIALIZE_MAP` boilerplate. It seems like this method is never used anywhere in the code. Is this simply outdated/unmaintained?

Reply:
@moneromooo-monero Could you comment on this?

Reply:
I don't/can't know if anyone is using them from third party software.

Reply:
OK, then I'll delete these lines and ignore them.

Reply:
"delete" as in remove the comments, or remove the indirection ?

Reply:
I mean, "remove the indirection" (i.e. not bothering with these (apparently) unused functions).

Reply:
The bother is tiny though, isn't it ? Why break things if leaving them unbroken is not only adding very little work, but you'd already done that work, albeit commented out ?

Reply:
They only cause a compile error due to the new code, right ?
submitblock is definitely used (by pools), so it should stay. getblockcount does seem redundant indeed. A middle ground is to keep those functions, but return an RPC error if proxying is enabled.

Reply:
> They only cause a compile error due to the new code, right ?

Yes, they cause a compiler error when I try to call `net_utils::invoke_http` with them.

> A middle ground is to keep those functions, but return an RPC error if proxying is enabled.

OK, I'll do something like:

```c++
if (should_use_bootstrap_daemon())
{
  res.status = "This RPC is unsupported for bootstrap daemon";
  return false;
}
```